### DISCUSSION TITLE
Synology DS218+

### DISCUSSION THREAD

**Discussion by simonmike86** [AdguardTeam/AdGuardHome#4589] [Q&A]
Have a question or an idea? Please search it [on our forum](https://github.com/AdguardTeam/AdGuardHome/discussions) to make sure it was not yet asked. If you cannot find what you had in mind, please [submit it here](https://github.com/AdguardTeam/AdGuardHome/discussions/new).

Hello I got this here always

<img width="1384" alt="image" src="https://user-images.githubusercontent.com/64546238/169774703-726841ac-3fd0-4137-acba-60e81b211324.png">

api.insight.synology.com
api.eu.c2.synology.com
api.insight.synology.com.fritz.box

what he want from synology? and look how much, over 29000x times

thank you

**Comment by ppfeufer** [comment]
That's your Synology NAS pinging home to check for updates. Like software updates for itself, app updates for the apps you have installed and so on.

**Comment by simonmike86** [comment]
ok because of so many requests, so its not bad?

When I’m activated Kids Security, I can’t deblock other sites?

Thanks

**Comment by ppfeufer** [comment]
> ok because of so many requests, so its not bad?

Having your NAS checking for updates is not a bad thing at all, in my opinion. The frequency depends on how your NAS and/or its software is configured. You might want to check the manual for more details on this or even ask Synology directly about it. Nothing the AGH team can do here I guess.



> When I’m activated Kids Security, I can’t deblock other sites?

Not sure what you mean by this, to be honest. You can configure AGH so that only certain clients use "Kids Security" and others don't.
